frontend/src/components/dashboard/create/telegram/TelegramRecipients.tsx

```diff
@@ -124,9 +124,8 @@ const TelegramRecipients = ({
       setCsvInfo((info) => ({ ...info, tempCsvFilename }))
     } catch (err) {
       setErrorMessage(err.message)
-    } finally {
```

Comment:
I originally thought this would be a noop, but it isn't.

The finally block will always execute even if there is a return statement within the try block.